server/datastore/mysql/activities.go

```diff
@@ -375,19 +381,21 @@ func (ds *Datastore) ListHostUpcomingActivities(ctx context.Context, hostID uint
 		fmt.Sprintf(`SELECT
 			hsi.execution_id as uuid,
 			-- policies with automatic installers generate a host_software_installs with (user_id=NULL,self_service=0),
-			-- thus the user_id for the upcoming activity needs to be the user that uploaded the software installer.
-			IF(hsi.user_id IS NULL AND NOT hsi.self_service, u2.name, u.name) AS name,
-			IF(hsi.user_id IS NULL AND NOT hsi.self_service, u2.id, u.id) as user_id,
-			IF(hsi.user_id IS NULL AND NOT hsi.self_service, u2.gravatar_url, u.gravatar_url) as gravatar_url,
-			IF(hsi.user_id IS NULL AND NOT hsi.self_service, u2.email, u.email) AS user_email,
+			-- so we mark those as "Fleet"
```

Comment:
Will this ever be "Fleet" for uninstalls? (There are no automatic uninstalls, right?)

Reply:
No automatic uninstalls at this point, so yeah, we won't see "Fleet uninstalled" ATM